bastion/gui/dashboard/main_window.py

```
@@ -0,0 +1,813 @@
+        self.timer.timeout.connect(self.refresh_status)
+        self.timer.start(3000)
+
+    def load_data(self):
+        # Load Rules
+        try:
+            if self.rules_path.exists():
+                with open(self.rules_path) as f:
+                    self.data_rules = json.load(f)
+            else:
+                self.data_rules = {}
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)